manifests/params.pp

```diff
@@ -29,9 +29,21 @@
   if $::operatingsystem == 'Ubuntu' {
     if versioncmp($::lsbdistrelease, '8.04') < 1 {
       $init_style = 'debian'
-    } else {
+    } elsif versioncmp($::operatingsystemrelease, '15.04') < 0 {
```

Comment:
I started programming puppet after version 3 and I've been a fan of less dependencies.  So I was always trying to convert our old stuff to operatingsystemrelease and operatingsystemmajrelease.  
lsbdistrelease is relying on redhat-lsb and whatever the package is in Ubuntu and other distros.  I believe the operatingsystem values are coming straight from facter Facter::Operatingsystem.implementation
Though i'm not sure where it gets that from.  I'm pretty sure the standing thought in puppet programming was to phase lsb out but that might be a personal preference of people...  as I said, I haven't been doing this long enough. :)